docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -151,3 +139,56 @@ fed-minion          <none>              Ready
 **The cluster should be running! Launch a test pod.**
 
 You should have a functional cluster, check out [101](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/README.md)!
+
+**Run time addition/deletion of minions (or nodes).**
+
+* Create following minion.json file on kubernetes master node:
+
+```json
+{
+  "id": "fed-node1",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+  "resources": {
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
+    },
+  },
+  "labels": {
+    "name": "fed-node1-label",
+  },
+}
+```
+
+Now create a minion object internally in your kubernetes cluster by running:
+
+```
+$ kubectl create -f minion.json
+
+$ kubectl get nodes
+NAME                LABELS              STATUS
+fed-node1           name=fed-node1-label     Unknown
```

Comment:
In my opinion, we should keep --machine stuff and so that a user is aware of it. The new method using json represetation is just another way so once they have setup a cluster using first method (using --machines), they can dive into the new one and that is why I put it after if they want to try further. 

I noticed that now minions (old terminology) are being called nodes (new terminology) so named it fed-node1, but I could change it to fed-minion-2, as fed-minion is already used with --machines.